packages/account/src/Components/poi/status/unsupported/card-details/utils.ts

```diff
@@ -31,13 +46,13 @@ export const validateFields = (values: FormikValues, fields = [], documents = []
         }
     });
 
-    documents.forEach((document: { name: string; label: string }) => {
-        const { name, label } = document;
+    documents.forEach((document: { name: string }) => {
+        const { name } = document;
         const value = values[name];
 
         if (checkIsEmpty(value)) {
             errors[name] = localize('{{label}} is required.', {
```

Comment:
The key used in interpolation (label) is different from the value (name)

Reply:
Yes coz the `label` is not even present in the `document` & the error is not being displayed anywhere, this is being used to set the error state if the document has been selected or not.